CHANGELOG.md

```diff
@@ -348,5 +348,6 @@ Released with 1.0.0-beta.37 code base.
 ### Added
 
 - Add `ETH2Core` class export to `index.d.ts` for `web3-eth2-core` (#3878)
+- Add `usage-examples` to simplify the usage of this package 
```

Comment:
Add the PR number please

Reply:
@koraykoska: Thanks for the hint --> done --> https://github.com/michael-spengler/web3.js/commit/2bc99a29e3a65ce8e6d37bfe9b673c1ab2cfb622#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed